controllers/aws_creds.go

```diff
@@ -24,8 +24,8 @@ const (
 	defaultProfile              = "saml"
 )
 
-func formatAccount(config *configreader.Config, login, role string) (*cfg.IDPAccount, error) {
-	timeDuration, err := time.ParseDuration(config.SessionDuration)
+func formatAccount(config *configreader.Config, duration, login, role string) (*cfg.IDPAccount, error) {
+	timeDuration, err := time.ParseDuration(duration)
```

Comment:
Are we going to check for invalid input and/or longevity threshold?

Reply:
longevity is checked by AWS itself and error returned there is quite meaningful